notification-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_channel.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.notification.config.service.v1;
+
+message NewNotificationChannel {
+  NotificationChannelMutableData notification_channel_mutable_data = 1;
+}
+
+message NotificationChannel {
+  string id = 1;
+  NotificationChannelMutableData notification_channel_mutable_data = 2;
+}
+
+message NotificationChannelMutableData {
```

Comment:
Note: I think, we will need changes in `notification-channels.json` accordingly in hypertrace-alert-engine, right? If, yes, can we log an issue for that as well?